packages/clay-alert/src/ClayToast.js

```diff
@@ -44,13 +44,14 @@ class ClayToast extends Component {
  */
 ClayToast.STATE = {
 	/**
-	 * Flag to indicate if alert should be automatically closed.
+	 * Flag to `true` to indicate whether the alert should be closed
+	 * automatically with the default time.
 	 * @default false
 	 * @instance
 	 * @memberof ClayToast
-	 * @type {?bool}
+	 * @type {?(bool|number)}
 	 */
-	autoClose: Config.bool().value(false),
+	autoClose: Config.oneOfType([Config.bool().value(false), Config.number()]),
```

Comment:
I think this format doesn't work, should be something like
 `Config.oneOfType([Config.bool()), Config.number()]).value(false)`

Reply:
Really, thanks for your observation!